x-pack/plugins/cases/server/saved_object_types/migrations.ts

```diff
@@ -59,6 +60,21 @@ interface SanitizedCaseType {
   type: string;
 }
 
+interface SanitizedCaseOwner {
+  owner: string;
+}
+
+const addOwnerToSO = <T = Record<string, unknown>>(
+  doc: SavedObjectUnsanitizedDoc<T>
+): SavedObjectSanitizedDoc<SanitizedCaseOwner> => ({
+  ...doc,
+  attributes: {
+    ...doc.attributes,
+    owner: SECURITY_SOLUTION_OWNER,
+  },
```

Comment:
Yeah I think we need this otherwise users who upgrade from 7.13 will not have an owner in their saved objects and I believe that will cause the rbac to consider the user not authorized.